README.md

```diff
@@ -4,4 +4,74 @@
 [![Container Repository on Quay](https://quay.io/repository/stackanetes/nova-kubernetes-drain/status "Container Repository on Quay")](https://quay.io/repository/stackanetes/nova-kubernetes-drain)
 [![Go Report Card](https://goreportcard.com/badge/stackanetes/nova-kubernetes-drain "Go Report Card")](https://goreportcard.com/report/stackanetes/nova-kubernetes-drain)
 
+The main goal of Nova-kubernetes-drain is to perform evacuation of [Openstack] compute node when [Kubernetes] node is being drained.
+Nova-kubernetes-drain should be deployed as a [Daemonset] via [Stackanetes].
 
+Nova-kubernetes-drain can be run as a daemon or as one-off task. Those two modes are simple configure by command line flag.
+
+## Requirements
+
+Nova-kubernetes-drain based on two clients:
+  1. [Kubernetes] client: https://godoc.org/k8s.io/kubernetes/pkg/client/unversioned
+  2. [Openstack] client*: http://gophercloud.io/docs/compute/
+
+  \* Rackspace client is deprecated, but new [client] currently does not support [live-migration]. Client should be switched when new client supports [live-migration]. 
+
+## Configuration file
+
+Nova-kubernetes-drain requires config.yaml. Configuration file should contain all variables necessary to establish connection with openstack.
+config.yaml example:
+
+```
+IdentityEndpoint: "http://keystone-api:5000/v3/"
+Username: "admin"
+Password: "mysupersecretpassword"
+TenantName: "admin"
+DomainID: "default"
```

Comment:
I wonder if we should mimic what OpenStack client supports? Environment variables or command-line parameters, named the same the than typical OpenStack clients. It's not a blocker at all though. More like a nit.